salvo/tools/coverage.sh

```diff
@@ -51,8 +62,11 @@ sed -e 's/SF.*\.runfiles\/salvo\/\(.*\)$/SF:\1/' -i coverage/salvo.dat
 # Generate HTML coverage report.
 mkdir -p coverage/html
 genhtml coverage/salvo.dat -o coverage/html
+zip -r coverage/html.zip coverage/html/
+
 echo "HTML coverage report generated, view by running:"
-echo "  (cd coverage/html && python3 -m http.server)"
+echo "  (cd salvo/coverage/html && python3 -m http.server)"
+echo "Or download the zip file from salvo/coverage/html.zip"
```

Comment:
Is this information still useful for developers? They can just see the log but cannot perform such operations.

Reply:
I think this is still useful for local executions. The `ci/do_ci.sh` script can also be executed locally to run tests, debug, etc.

I have updated the text in 1f3f2466a226e346a80e8d57b7664c0d1e1ef989 to clarify with regards to the two execution modes.